purchases/src/main/kotlin/com/revenuecat/purchases/strings/PurchaseStrings.kt

```diff
@@ -8,7 +8,8 @@ internal object PurchaseStrings {
     const val CONSUMING_PURCHASE = "Consuming purchase with token %s"
     const val CONSUMING_PURCHASE_ERROR = "Error consuming purchase. Will retry next queryPurchases. %s"
     const val NOT_CONSUMING_IN_APP_PURCHASE_ACCORDING_TO_BACKEND = "Not consuming in-app purchase according to" +
-        " server configuration."
+        " server configuration. This is expected for non-consumable products. User won't be able to purchase this" +
+        " product again."
```

Comment:
Not sure how these are surfaced, but maybe: 
 ```suggestion
        " server configuration. This is expected for non-consumable products. Users won't be able to purchase this" +
        " product again."
``` 
Or:
```suggestion
        " server configuration. This is expected for non-consumable products. The user won't be able to purchase this" +
        " product again."
```
What do you think?

Reply:
These are logcat logs, as long as the developer has set a specific `LogLevel` like this: https://www.revenuecat.com/docs/test-and-launch/debugging. And yeah, I think that's fine 👍

Reply:
Ah thanks for the context!